packages/auth/src/grant/routes.test.ts

```diff
@@ -170,54 +170,57 @@ describe('Grant Routes', (): void => {
         scope.done()
       })
 
-      test('Can get a software-only authorization grant', async (): Promise<void> => {
-        const ctx = createContext<CreateContext>(
-          {
-            headers: {
-              Accept: 'application/json',
-              'Content-Type': 'application/json'
+      test.each`
+        accessType
+        ${AccessType.IncomingPayment}
+        ${AccessType.Quote}
+      `(
+        'Can get an grant without interaction for $accessType grants',
```

Comment:
we don't generally have a good pattern for testing different ENV variable configurations. Should I recreate a new container with different Config variables to test the cases where interaction for incoming payments and quotes _is_ enabled?

Reply:
We dabbled with that in
- https://github.com/interledger/rafiki/pull/683

but ended up relocating the env var / config check to outside the scope of the relevant tests
- https://github.com/interledger/rafiki/pull/735

Could we similarly move this check outside the route? To a prior middleware?